docs/walk-through/loops.md

```diff
@@ -69,7 +79,7 @@ spec:
       args: [/etc/os-release]
 ```
 
-We can pass lists of items as parameters:
+This example does exactly the same job as the previous example, but using `withParam` to pass the information as a JSON array argument, instead of directly hardcoded into the template.
```

Comment:
perhaps "...instead of being directly hardcoded..."? or "hard-coding"?

Reply:
Done.